src/createAsyncThunk.ts

```diff
@@ -173,9 +201,17 @@ export function createAsyncThunk<
                 getState,
                 extra,
                 requestId,
-                signal: abortController.signal
```

Comment:
Can you add that notional `AbortController` pseudo-polyfill in this PR as well?

Reply:
Gonna do that in an extra PR in the next few days, I think this has been blocked long enough :)